README.md

```diff
@@ -75,15 +75,24 @@ Note: For the different image-types you have to use special --ovh-ssh-user (for
 |``--ovh-image``                                            |Cloud Machine image|Ubuntu 16.04 |no|
 |``--ovh-ssh-user``                                         |Cloud Machine SSH User|ubuntu |no|
 |``--ovh-project``                                          |Cloud Project name/description or id|single one|only if multiple projects|
```

Comment:
I think this line should be removed. Looks like an older version of the line below